analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -150,6 +165,18 @@ private void writeMarkerFile(File rootDir) throws IOException {
         return result;
     }
 
+    private Set<SimpleProjectRef> getUnresolvedDependenciesExcludingProjectDependencies(LenientConfiguration lenient,
+            Set<ProjectRef> allProjectModules) {
+        return lenient.getUnresolvedModuleDependencies()
+                .stream()
+                .map(d -> {
+                    final ModuleVersionSelector selector = d.getSelector();
+                    return new SimpleProjectRef(selector.getGroup(), selector.getName());
+                })
+                .filter(p -> !allProjectModules.contains(p))
```

Comment:
If we are passing in Set<String> projectDependencies then I think this could be made simpler?

Reply:
I thought so initially as well, but `Dependency` does not related to `UnresolvedDependency`, so you can't really use them for comparison (unless I missed something)

Reply:
Can we use the moduleversionselector to compare with the projectdependency group:name:version?

Reply:
UnresolvedDependency.getSelector?

Reply:
Searched for something, came up empty :(